tests/acceptance/features/apiSpaces/uploadSpaces.feature

```diff
@@ -0,0 +1,50 @@
+@api @skipOnOcV10
+Feature: Upload files into a space
+  As a user
+  I want to be able to work with project spaces and quota
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "Bob" has been created with default attributes and without skeleton files
+
+  Scenario: Alice creates a folder via Graph api and uploads a file
+    Given the administrator gives "Alice" the role "Admin" using the settings api
+    When user "Alice" creates a space "Project Moon" of type "project" with quota "2000" using the GraphApi
+    And user "Alice" lists all available spaces via the GraphApi
+    And user "Alice" creates a folder "NewFolder" in space "Project Moon" using the WebDav Api
+    Then the HTTP status code should be "201"
```

Comment:
we have same test `Alice creates a folder via the Graph api in space, she expects a 201 code and she checks that folder exists` in ListSpaces. or did you want upload file into folder?

Reply:
upload into folder is possible. But currently we cannot list a propfind with Depth > 1